packages/Path-View-Core.package/SClassDiagram.class/instance/extendDiagramAreaTo..st

```diff
@@ -0,0 +1,10 @@
+geometry
+extendDiagramAreaTo: newBounds
+
+	| bounds |
+	bounds := self boundsInWorld.
+	(bounds top > newBounds top) ifTrue: [self dragTopBy: (bounds top - newBounds top)].
```

Comment:
we have to do the check, because we do not want to shrink the diagram again. this makes thinks complicated. though after the check one could possibly just assign the new top value. @AlexImmer what do you say, you built this.